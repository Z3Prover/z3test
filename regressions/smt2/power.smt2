(declare-const a Int)
(declare-const b Int)

(simplify (<= (+ (* a a) (* b b b)) -1))
(simplify (<= (+ (* a a) (* b b b b)) -1))
(simplify (<= (+ (* a a) (* b b b b)) 0))
(simplify (>= (+ (* -1 a a) (* -2 b b)) 1))
(simplify (>= (+ (* -1 a a) (* -2 b b)) 0))
