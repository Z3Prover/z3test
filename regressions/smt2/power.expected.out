(<= (+ (* a a) (* b b b)) (- 1))
false
(and (= a 0) (= b 0))
false
(and (= a 0) (= b 0))
